eng/WpfArcadeSdk/tools/GenApi/GlobalApiExclusions.txt

```diff
@@ -0,0 +1,23 @@
+// NOTE: 
+//   The following commented out removals are due to issues with generating derived classes.
```

Comment:
Is APICompat able to use the same baselines for ref and runtime assemblies?

Reply:
Runtime assemblies were never baselined against .NET 4.8.  We picked up the compiler generated ref output and baselined that.  The filename just had the runtime assembly name.  In any case I need to clean those files and remove the runtime assemblies from the list of apicompat runs.

Reply:
I actually found some bugs in the ApiCompat implementation and I'm working them out right now.